virtwho/virt/rhevm/rhevm.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, logger, config, dest, terminate_event=None,
                                     interval=interval,
                                     oneshot=oneshot)
         self.url = self.config.server
+        self.api_base = 'api/'
```

Comment:
I think you should leave trailing slash off of `self.api_base`, and instead put `/` as a prefix on `{clusters,hosts,vms}_endpoint`...